lib/src/browser/disposable_browser.dart

```diff
@@ -198,14 +199,20 @@ class Disposable implements disposable_common.Disposable {
   void manageDisposable(disposable_common.Disposable disposable) =>
       _disposable.manageDisposable(disposable);
 
+  @deprecated
   @override
   void manageDisposer(disposable_common.Disposer disposer) =>
```

Comment:
Do you see the doc comment inherited from the interface in your IDE @aaronstgeorge-wf ?